tests/test_black.py

```diff
@@ -114,7 +116,10 @@ class BlackRunner(CliRunner):
     """Make sure STDOUT and STDERR are kept separate when testing Black via its CLI."""
 
     def __init__(self) -> None:
-        super().__init__(mix_stderr=False)
+        if Version(imp_version('click')) >= Version('8.2.0'):
```

Comment:
```suggestion
        if Version(imp_version("click")) >= Version("8.2.0"):
```
CI is unhappy because the quotes get reformatted to double quotes, but I think it should be gtg after this.

Reply:
ok